lib/onelogin/ruby-saml/logoutresponse.rb

```diff
@@ -69,46 +104,211 @@ def status_code
         end
       end
 
+      # @return [String] Gets the StatusMessage from a SAML Response.
+      #
+      def status_message
+        @status_message ||= begin
+          node = REXML::XPath.first(document, "/p:LogoutResponse/p:Status/p:StatusMessage", { "p" => PROTOCOL, "a" => ASSERTION })
+          node.text if node
+        end
+      end
+
+      # Gets the expected current_url
+      # (Right now we read this url from the Sinle Logout Service of the Settings)
+      # TODO: Calculate the real current_url and use it.
```

Comment:
@phlipper do you have any general get_current_url method?

In python toolkit we use https://github.com/onelogin/python-saml/blob/master/src/onelogin/saml2/utils.py#L320 that depends on the request_data and each framework prepare this value:
https://github.com/onelogin/python-saml/blob/master/demo-django/demo/views.py#L17

In PHP is easier since the values to build the current_url are at the global $_SERVER var: https://github.com/onelogin/php-saml/blob/master/lib/Saml2/Utils.php#L340

A first approach is to check that the 'Destination' value match the ACS URL, later I want to validate that the real destination match the 'Destination' value.